docs/changelog.md

```diff
@@ -12,6 +12,9 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Config.kt)
 
+* **feature-addons**
+    * AddonManager now receives a WebExtensionRutime instance instead of an Engine.
```

Comment:
nit: WebExtensionRu[n]time.

We can also point out that this doesn't not require changes for consumers (as Engine already implements the interface).

Reply:
Fixed.